kubernetes/docker_testing.py.j2

```diff
@@ -0,0 +1,15 @@
+from .testing import *
+
+DATABASES = {
+    'default': {
+        # NAME intentionally left unset, in case tests try to access the production database.
+        # This could happen e.g. if someone executes a query at module level.
+        'ENGINE':          'django.db.backends.postgresql_psycopg2',
+        'USER':            'postgres',
+        'HOST':            '{{ image_prefix }}postgresql-unittest',
+        'ATOMIC_REQUESTS': True,
+        'TEST': {
+            'NAME': 'test_concent_api'
+        }
+    }
+}
```

Comment:
Instead of replacing the whole `DATABASES` setting, only change values in the default config.

Also, now that Conductor has been merged, please configure its database too.

Reply:
Done.